documentation/airport-guides/lrop.md

```diff
@@ -0,0 +1,36 @@
+### About this facility
+Henri Coanda International Aiport (LROP / OTP), Otopeni, Bucharest, Romania, EU
+116,718 annual operations in 2017 ([WIKIPEDIA](https://en.wikipedia.org/wiki/Henri_Coand%C4%83_International_Airport#Passengers))
+
+Located at about 7 miles north of Bucharest, the capital of Romania, it is the main international airport in the country. It has become the busiest airport since Baneasa Aurel Vlaicu Airport was converted into a general aviation airport and all low cost operations were moved to Otopeni. Traffic consists of medium aircraft (mostly from A320 and B737 families) with seldom heavy arrivals.
+
+The airport has two parallel runways 08R/L, 26L/R each of 11483ft/3500m. ILS CAT III A/B is available at both 08 runways and ILS CAT I at runways 26.
+
+The airport is the main hub for TAROM (Romania's national airline operator) and Blue Air (the first low cost Romanian operator) and also hosts a military apron used by the Romanian Air Force.
+
+### Airspace
+Bucharest Terminal Movement Area is a class A airspace, equivalent to American class B, starting from 2000 feet and ending at FL175, spreading out about 30 nm in each cardinal direction. Transition altitude is 4000 feet.
+
+Airspace above Bucharest is restricted (LRR3) up to FL105, entering it requires special permission from the local authorities. Clinceni danger area (LRD100) is temprary activated during the summer with parachute jumping activity up to FL130.
```

Comment:
temprary -> temporarily

Reply:
Airspace above Bucharest is restricted (LRR3) up to FL105; entering it requires special permission from the local authorities.

Semicolon instead of comma